tests/playwright/src/specs/compose-onboarding-smoke.spec.ts

```diff
@@ -140,7 +158,30 @@ test.describe.serial('Compose onboarding workflow verification', { tag: '@smoke'
     const cliToolsPage = await settingsBar.openTabPage(CLIToolsPage);
     const composeRow = cliToolsPage.toolsTable.getByLabel(RESOURCE_NAME);
     const composeVersionInfo = composeRow.getByLabel('cli-version');
-    await playExpect(composeVersionInfo).toHaveText('docker-compose ' + composeVersion);
+    await playExpect(composeVersionInfo).toContainText('docker-compose ' + composeVersion);
+  });
+
+  test('Check Podman Desktop autorefresh when using podman compose up', async ({ navigationBar }) => {
+    test.setTimeout(300_000);
+
+    const composeFilePath = path.resolve(__dirname, '..', '..', 'resources', `compose.yaml`);
+    await runComposeUpFromCLI(composeFilePath);
+
+    const containersPage = await navigationBar.openContainers();
+    await playExpect(containersPage.heading).toBeVisible();
+
+    await playExpect
```

Comment:
Looks good, just part of the test should be a check for compose tagged container (kinda parent container containing these two). 
![Screenshot_20250115_125224](https://github.com/user-attachments/assets/b1ffddd1-3fc9-4fce-802b-d33d76cc7327)
This way user knows it is compose created container.